Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12-alpine
+LABEL authors="user"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR app
```

Comment:
The `WORKDIR` command should specify an absolute path. Consider changing `WORKDIR app` to `WORKDIR /app` to ensure the working directory is correctly set.